src/components/breadcrumbs/examples/BreadcrumbsExamples.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Breadcrumb } from '../Breadcrumb';
+import { ITitleProps } from '../../title/Title';
+import { ILinkSvgProps } from '../../svg/LinkSvg';
+import { IBreadcrumbLinkProps } from '../BreadcrumbLink';
+
+export class BreadcrumbsExamples extends React.Component<any, any> {
+
+  render() {
+    const documentationLink: ILinkSvgProps = {
+      url: 'https://www.google.ca',
+      target: '_blank',
+      svg: {
+        svgName: 'help',
+        svgClass: 'fill-orange icon mod-20',
+      },
+    };
+
+    const defaultTitle: ITitleProps = {
+      title: 'Pokemon',
+      prefix: 'Catch them all:',
+      documentationLink,
+    };
+
+    const link1: IBreadcrumbLinkProps = {
+      name: 'Pikachu',
+      link: '#',
```

Comment:
consider adding real paths here, `https://www.google.ca/?q=pokemon` `https://www.google.ca/?q=pokemon/squirtle` , it's nice to have demos that do something

Reply:
it could be nice to have an example with a defaultLinkPath, following the example above it would be `https://www.google.ca/?q=` as a default path link and your first breadscrum link would be `pikachu/` and your second one `squirtle`

Reply:
I know it won't be different in the examples, but it's a working example we can look at when about to use the component, they are examples but we should see it as documentation at the same time i think

Reply:
added